dockers/docker-pde/base_image_files/port_breakout.py

```diff
@@ -0,0 +1,833 @@
+#! /usr/bin/python -u
+
+import os
+import sys
+import re
+import getopt
+import copy
+import json
+import shutil
+import os.path
+import subprocess
+
+SIM_HOST = False
+
+SAI_PROFILE_DELIMITER = '='
+INTERFACE_KEY="Ethernet"
+if not SIM_HOST:
+    NEW_FILE_EXT=""
+else:
+    NEW_FILE_EXT=""
```

Comment:
Why the need for this if/else statement if the resulting file extension is the same (empty) for both cases?